### PR TITLE
Updated the Glossary article

### LINKED ISSUE #4846: Update the GitHub glossary article
### What article on docs.github.com is affected?

"[GitHub glossary](https://docs.github.com/en/github/getting-started-with-github/github-glossary)"

### What part(s) of the article would you like to see updated?

- Improve the intro to make it friendlier to new users and use less awkward phrasing.
- Remove capitalization from these headers
  - check
  - code frequency graph
  - contributors graph
  - dashboard
  - dependents graph
  - dependencies graph (also change to “dependency graph”)
  - enterprise account
  - gist
  - management console
  - markup
  - members graph
  - network graph
  - news feed
  - pulse graph
  - punch graph
  - status checks
  - traffic graph
- Delete this entry
  - Works with GitHub

### Additional information

This issue is for a special project and not currently available for contributors to work on.

### PR DESCRIPTION
**Pull request by codewithdev**
### Why:

Closes #4846 #4844

### What's being changed:
- This PR updates the Glossary Introductory Lines.
- Removed Capitalization from the terms listed below,
     - check
     - code frequency graph
     - contributors graph
     - dashboard
     - dependents graph
     - dependencies graph (also change to “dependency graph”)
     - enterprise account
     - gist 
     - management console
     - markup
     - members graph
     - network graph
     - news feed
     - pulse graph
     - punch graph
     - status checks
     - traffic graph

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by janiceilene — COMMENTED** [review]
The current changes are looking great @codewithdev 🎉 There's one more item on the issue to do. 

> Improve the intro to make it friendlier to new users and use less awkward phrasing.

Do you want to take a pass at improving the intro, then ping me back in so I can triage this one for review? Thanks!

**Comment by codewithdev** [comment]
I had added introductory lines too but it seems the PR has been affected for some reason. @janiceilene I will work further to improve the Glossary Introdutory Lines.

**Comment by janiceilene** [comment]
🎉 Now I'm seeing those changes! I'll get this triaged for review ⚡

File: content/github/getting-started-with-github/quickstart/fork-a-repo.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ topics:
 ---
 ### About forks
 
-Most commonly, forks are used to either propose changes to someone else's project or to use someone else's project as a starting point for your own idea.
+Most commonly, forks are used to either propose changes to someone else's project or to use someone else's project as a starting point for your own idea. You can fork a repository to create a copy of the repository and make changes without affecting the upstream repository. For more information, see "[Working with forks](/github/collaborating-with-issues-and-pull-requests/working-with-forks)."
```
Comment (ethanpalm) [inline content/github/getting-started-with-github/quickstart/fork-a-repo.md:21]: I made a few changes to the intro here and moved the link to "[Working with forks](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks)" to the intro here.

In the GitHub docs, we don't use headers for links so I was looking for another place to put the link and it seems to make sense here if people have more questions about what to do with forks when they're reading the beginning of this article.

Let me know if you have any questions about these changes ✏️

File: content/github/getting-started-with-github/quickstart/github-glossary.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: GitHub glossary
-intro: 'Below are a list of some Git and {% data variables.product.prodname_dotcom %} specific terms we use across our sites and documentation.'
+intro:  'When working with Git and GitHub, you may find Technical Terms that powers the {% data variables.product.prodname_dotcom %} sites and its documentation. We can find the definition or meaning of these words directly from the Glossary.'
```
Comment (ethanpalm) [inline content/github/getting-started-with-github/quickstart/github-glossary.md:3]:
```suggestion
intro:  'This glossary introduces common Git and {% data variables.product.prodname_dotcom %} terminology.'
```

File: data/glossaries/external.yml [hunk 3]
```diff
@@ -175,17 +175,17 @@
...
 - term: default branch
   description: >-
     The base branch for new pull requests and code commits in a repository. Each repository has at least one branch, which Git creates when you initialize the repository. The first branch is usually called {% if currentVersion ver_lt "enterprise-server@3.2" %}`master`{% else %}`main`{% endif %}, and is often the default branch.
-- term: Dependents graph
+- term: dependents graph
   description: >-
     A repository graph that shows the packages, projects, and repositories that depend on a
     public repository.
-- term: Dependencies graph
+- term: dependencies graph
```
Comment (ethanpalm) [inline data/glossaries/external.yml:188]:
```suggestion
- term: dependency graph
```

**Review by ethanpalm — CHANGES_REQUESTED** [review]
Thanks for opening a PR to fix these issues, @codewithdev!

I've made a few suggestions. Please let me know if you have any questions about them. Once those changes are made, I can update and merge this branch and we'll get these changes live on the docs site 🎉

File: content/github/getting-started-with-github/quickstart/fork-a-repo.md (hunk 1 above)
Comment (codewithdev) [reply content/github/getting-started-with-github/quickstart/fork-a-repo.md:21]: That certainly make sense.

**Review by codewithdev — COMMENTED** [review]
Updated the requested Changes 👍

**Review by ethanpalm — APPROVED** [review]
These changes look good to go 🚀  I'm going to update the branch and I'll merge it once all the checks are passing. The changes should be live on the docs site within 24 hours ⚡

**Pull request merged**